helm_deploy/manage-recalls-api/templates/networkpolicy-ppud-replacement-ingess.yaml

```diff
@@ -18,4 +18,4 @@ spec:
     - from:
       - namespaceSelector:
           matchLabels:
-            component: ppud-replacement-dev
+            cloud-platform.justice.gov.uk/namespace-name: ppud-replacement-dev
```

Comment:
Is `ppud-replacement-...` correct? I thought everything was `manage-recalls-...`?

Reply:
Ah, this is to allow ingress *from* ppud-replacement-dev, which is where the dashboard app lives.